README.md

```diff
@@ -6,3 +6,52 @@
 [![Coverage](https://codecov.io/gh/invenia/FeatureDescriptors.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/invenia/FeatureDescriptors.jl)
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
 [![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
+
+FeatureDescriptors.jl is an interface package for describing features used in [models](https://invenia.github.io/Models.jl/stable/), [feature engineering](https://invenia.github.io/FeatureTransforms.jl/stable/), and other data-science workflows.
+
+## Getting Started
+
+A `Descriptor` is an abstract type that provides a way to describe the properties and relationships for a collection of features at a high-level.
+`Descriptor` supertypes define some general properties, such as the required tables or value column, that can be overloaded for specific
```

Comment:
"supertypes" sounds like a misnomer to me because `Descriptor` is at the top - isn't it "abstract subtypes" vs. "concrete subtypes"?

Reply:
A concrete subtype would be a struct. "Supertype" is the correct term. A supertype can be a subtype of another type.

Reply:
Oh yep, I was mistaken about abstract vs. concrete. But "`Descriptor` supertypes" sounds like supertypes _of_ `Descriptor`, like they are above `Descriptor` in the hierarchy, when they are actually below.

Reply:
This could probably be reworded to be less ambiguous. 

"Some Descriptor subtypes are supertypes of other Descriptors that define some general properties..."?

Reply:
I'll word it better.